app/main.py

```diff
@@ -2,0 +1,17 @@
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    mask_count = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(visitor=friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
```

Comment:
Code Style Issue: It's better to use a more descriptive error message. For example, you can include the name of the friend who is not vaccinated in the error message. This way, it will be easier to understand who exactly has the problem.